.azure-pipelines/scripts/validate_samples.R

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env
+args <- commandArgs(trailingOnly = TRUE)
```

Comment:
Most of our samples will call load_workspace_config(). How do we handle config.json?

Reply:
added write_workspace_config before running samples